includes/UserAdd.php

```diff
@@ -141,7 +141,7 @@
 	//
 	$userdb = str_replace('.','$',$user) ;
 	
-	$stmt = $sqlConn->prepare("CREATE TABLE $userdb ( $UserAppId BIGINT UNSIGNED NOT NULL, $UserAppTy VARCHAR(255), INDEX idx USING BTREE($UserAppId), INDEX idx1 USING BTREE($UserAppTy) )" );
+	$stmt = $sqlConn->prepare("CREATE TABLE $userdb ( $UserAppId BIGINT UNSIGNED NOT NULL, $UserAppTy VARCHAR(255), $AppDate VARCHAR(10) NOT NULL, INDEX idx USING BTREE($UserAppId), INDEX idx1 USING BTREE($UserAppTy) , INDEX idx2 USING BTREE($AppDate) )" );
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 257 characters

<!-- Comment by [SideCI](https://sideci.com) -->